go/vt/sqlparser/normalizer_test.go

```diff
@@ -75,14 +75,14 @@ func TestNormalize(t *testing.T) {
 	}, {
 		// float val
 		in:      "select * from t where foobar = 1.2",
-		outstmt: "select * from t where foobar = :foobar /* DECIMAL */",
+		outstmt: "select * from t where foobar = :foobar /* DECIMAL(2,1) */",
```

Comment:
@GuptaManan100 Been testing this, but it looks like `datetime` / `time` have a similar problem. We need to store the precision there too.

Reply:
@GuptaManan100 Pushed up a similar change for `datetime` & `time` to ensure we keep the precision there too.

Reply:
Sweet! Thank you!